embeddings/data/datamodule.py

```diff
@@ -25,6 +25,10 @@
 class BaseDataModule(abc.ABC, pl.LightningDataModule, Generic[Data]):
     dataset: Data
 
+    def __init__(self) -> None:
+        super().__init__()  # type: ignore
```

Comment:
```suggestion
    def __init__(self) -> None:
        # ignoring the type to avoid calling to untyped function "__init__" in typed context error
        # caused by pl.LightningDataModule __init__ method not being typed
        super().__init__()  # type: ignore
```